doc/stdlib/parallel.rst

```diff
@@ -253,13 +265,23 @@ General Parallel Computing Support
 
    Get the id of the current process.
 
-.. function:: pmap(f, lsts...; err_retry=true, err_stop=false, pids=workers())
+.. function:: asyncmap(f, c...) -> collection
+
+   .. Docstring generated from Julia source
+
+   Transform collection ``c`` by applying ``@async f`` to each element.
+
+   For multiple collection arguments, apply f elementwise.
+
+.. function:: pmap([::WorkerPool], f, c...) -> collection
 
    .. Docstring generated from Julia source
 
-   Transform collections ``lsts`` by applying ``f`` to each element in parallel. (Note that ``f`` must be made available to all worker processes; see :ref:`Code Availability and Loading Packages <man-parallel-computing-code-availability>` for details.) If ``nprocs() > 1``\ , the calling process will be dedicated to assigning tasks. All other available processes will be used as parallel workers, or on the processes specified by ``pids``\ .
+   Transform collection ``c`` by applying ``f`` to each element using available workers and tasks.
+
+   For multiple collection arguments, apply f elementwise.
 
-   If ``err_retry`` is ``true``\ , it retries a failed application of ``f`` on a different worker. If ``err_stop`` is ``true``\ , it takes precedence over the value of ``err_retry`` and ``pmap`` stops execution on the first error.
+   Note that ``f`` must be made available to all worker processes; see :ref:`Code Availability and Loading Packages <man-parallel-computing-code-availability>` for details.
```

Comment:
A mention about `@catch` and `retry` and the functionality they provide (as replacement for `err_retry` and `err_stop`) will help.